docs/api/registration-base-url-resource.md

```diff
@@ -216,6 +217,26 @@ registration | string | no       | The URL to the registration index for this de
 If the `range` property is excluded or an empty string, the client should default to the version range `(, )`. That is,
 any version of the dependency is allowed.
 
+#### Package deprecation
+
+Each package deprecation has the following properties:
+
+Name             | Type             | Required | Notes
+---------------- | ---------------- | -------- | -----
+reasons          | array of strings | yes      | The reasons why the package was deprecated
+message          | string           | no       | The additional details about this deprecation
+alternatePackage | object           | no       | The package dependency that should be used instead
+
+The `reasons` property should only contains strings from the following table:
+
+Reason       | Description             
+------------ | -----------
+Legacy       | The package is no longer being maintained
+CriticalBugs | The package has bugs which make it unsuitable for usage
+Other        | The package was deprecated due to a reason not on this list
+
+If the `reasons` property contains strings that are not from the known set, they should be ignored. Additionally, if the property contains only strings that are not from the known set, it should be treated as if it only contained the "Other" string.
```

Comment:
Does that mean that empty array of reasons should be treated as `["Other"]`? Should we call that out explicitly?

Reply:
Empty array should not be a valid state.